MyHashMap.java

```diff
@@ -0,0 +1,40 @@
+// Time Complexity : O(1)
+// Space Complexity :O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : is time complexity right for this problem i am a bit confused.
```

Comment:
No, it should be O(n) since Array.fill uses O(n) time

Reply:
Is the space complexity correct for this solution, even if I initialize  the array with the length of 1 billion the space complexity would still be O(1)?